package-lock.json

```diff
@@ -4430,40 +4430,6 @@
         }
       }
     },
-    "bpmn-js": {
```

Comment:
Is this a necessary change? If not, let us remove it.

Reply:
:+1: Done